hbase-handler/src/test/results/negative/cascade_dbdrop.q.out

```diff
@@ -18,8 +18,7 @@ TBLPROPERTIES ("hbase.table.name" = "hbase_table_0", "external.table.purge" = "t
 POSTHOOK: type: CREATETABLE
 POSTHOOK: Output: database:hbasedb
 POSTHOOK: Output: hbaseDB@hbase_table_0
-Found 3 items
-drwxr-xr-x   - ### USER ### ### GROUP ###          0 ### HDFS DATE ### hdfs://### HDFS PATH ###
+Found 2 items
```

Comment:
what is this change? One entry got missing?

Reply:
Here i did a quick local test using hbase2.2.4 and hbase2.5.6:
I create a hbase table using **Hbase shell** command cli:
```
hbase:001:0> create 'mytable', 'cf1', 'cf2'
Created table mytable
```


- I checked the hdfs data directory of the table in Hbase 2.2.4 and found it has three directory
```
Found 3 items
drwxr-xr-x   - root supergroup          0 2023-10-29 16:16 hdfs://localhost:8028/hbase224/data/default/mytable/.tabledesc
drwxr-xr-x   - root supergroup          0 2023-10-29 16:16 hdfs://localhost:8028/hbase224/data/default/mytable/.tmp
drwxr-xr-x   - root supergroup          0 2023-10-29 16:16 hdfs://localhost:8028/hbase224/data/default/mytable/556dcd44aa1534942170a780383ad2d3
```
- I checked the hdfs data directory of the table in Hbase 2.5.6 and found it only has two directory
```
 Found 2 items
drwxr-xr-x   - root supergroup          0 2023-10-29 16:44 hdfs://localhost:8028/hbase256/data/default/mytable/.tabledesc
drwxr-xr-x   - root supergroup          0 2023-10-29 16:44 hdfs://localhost:8028/hbase256/data/default/mytable/50cac0c44862cd5f9e8387677b0cc0ad
```

It is quite clear that new Hbase version(2.5.6) has changed its data directory structure, and this This change has nothing to do with Apache Hive.

Do you think my this test can prove that this change has nothing to do with Apache Hive?

Reply:
Looks fine, Will merge later today